src/vaadin-chart.html

```diff
@@ -1023,7 +1036,7 @@
          * Add an axis with given options
          *
          * @param {Object} options axis options
-         * @param {Boolean} isXAxis indicates if axis is X (`true`) or Y (`false`). Defaults to `false`.
+         * @param {boolean} isXAxis indicates if axis is X (`true`) or Y (`false`). Defaults to `false`.
          */
         __addAxis(options, isXAxis) {
```

Comment:
`__addAxis()` is called from vaadin-chart-series so should be `@protected`?

Reply:
Done.